cql3/expr/expression.cc

```diff
@@ -603,14 +611,32 @@ value_list get_IN_values(const ::shared_ptr<term>& t, size_t k, const query_opti
         const auto val = static_pointer_cast<tuples::in_value>(mkr->bind(options));
         const auto split_values = val->get_split_values(); // Need lvalue from which to make std::view.
         const auto result_range = split_values
-                | boost::adaptors::transformed([k] (const std::vector<bytes_opt>& v) { return v[k]; }) | non_null | deref;
+                | boost::adaptors::transformed([k] (const std::vector<managed_bytes_opt>& v) { return v[k]; }) | non_null | deref;
         return to_sorted_vector(std::move(result_range), comparator);
     }
     throw std::logic_error(format("get_IN_values(multi-column) on invalid term {}", *t));
 }
 
 static constexpr bool inclusive = true, exclusive = false;
 
+/// A range of all X such that X op val.
+nonwrapping_range<managed_bytes> to_range(oper_t op, managed_bytes&& val) {
```

Comment:
Did you try modifying the template version (on line 678) to take a moved value?  That would be preferable to adding this near-identical overload.

Reply:
Done.